cfme/infrastructure/config_management.py

```diff
@@ -250,7 +251,7 @@ def load_from_yaml(cls, key):
             name=data['name'],
             url=data['url'],
             ssl=data['ssl'],
-            credentials=cls.Credential(
+            credentials=cfme.base.credentials.Credential(
```

Comment:
**REQUIRED:** This should still be ```cls.``` I think as it uses the providers ```Credential``` object, not the base one